### PR TITLE
change setup to cfg format

### PR DESCRIPTION
**Pull request by jrbalderrama**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    wheel>=0.31.0
+install_requires =
+    ansible>=2.5.0,<2.6.0
+    jsonschema >= 2.6.0, < 2.7
+    execo >= 2.6.2, < 2.7
+    requests>=2.18.0, <2.19
+    python-vagrant>=0.5.15
+    netaddr>=0.7,<0.8    
+
+[options.extras_require]
```
Comment (msimonin) [inline setup.cfg]: 👍 
Maybe need to update the doc at some point.

Comment (jrbalderrama) [reply setup.cfg]: done ;)

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    python-novaclient<10
+    python-neutronclient==6.3.0
+    python-blazarclient==1.0.1
+
+[options.packages.find]
+exclude =
+    docs
+
+[bdist_wheel]
+python-tag = py27
```
Comment (msimonin) [inline setup.cfg]: we support python3
Should we go for `universal = 1`

**Comment by jrbalderrama** [comment]
each time you update on your side the code this PR has conflicts because I propose the configuration on the CFG file not in Python code

**Comment by msimonin** [comment]
Thanks let's wheel !

**Pull request merged**